.github/workflows/greeting.yml

```diff
@@ -0,0 +1,33 @@
+name: Greeting
+on:
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  greeting:
+    name: Issue a greeting comment upon PR
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Issue the greeting comment
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          token: ${{ secrets.CI_TOKEN }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            Hi @${{ github.event.pull_request.user.login }}. Thanks for your PR!
+
+            I am @adamjensenbot.
+            You can interact with me issuing a **slash command** in the first line of a **comment**.
+            Currently, I understand the following commands:
+            * `/rebase`:            Rebase this PR onto the master branch
+            * `/merge`:             Merge this PR into the master branch
+            * `/ok-to-test`         Enable testing for PRs with base on forks
+
+            Make sure this PR appears in the **${{ github.event.repository.name }} changelog**, adding one of the following **labels**:
+            * `kind/breaking`:      :boom: Breaking Change
+            * `kind/feature`:       :rocket: New Feature
+            * `kind/bug`:           :bug: Bug Fix
+            * `kind/cleanup`:       :broom: Code Refactoring
+            * `kind/docs`:          :memo: Documentation
```

Comment:
As you are changing the labels, it is also required to update the `lerna.json` file for the changelog generation.